### PR TITLE
Add SharedWorker support

### PR DESCRIPTION
**Pull request by saschanaz**
I went the lazy way and just copied runWorker, but I should probably merge those two. Or do you want this as-is, as it seems runServiceWorker is also quite duplicated?

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -17,6 +17,7 @@
       <option value="">All Global Exposures</option>
       <option value="?exposure=Window">Window</option>
       <option value="?exposure=Worker">Worker</option>
```
Comment (vinyldarkscratch) [inline views/index.ejs:19]: Since we're here, we might as well specify this is a dedicated worker?

```suggestion
      <option value="?exposure=Worker">Dedicated Worker</option>
```

File: views/index.ejs [hunk 2]
```diff
@@ -17,6 +17,7 @@
...
+      <option value="?exposure=SharedWorker">SharedWorker</option>
```
Comment (vinyldarkscratch) [inline views/index.ejs:20]:
```suggestion
      <option value="?exposure=SharedWorker">Shared Worker</option>
```

File: static/resources/serviceworker.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC
+// Copyright 2020 Mozilla Corporation
```
Comment (vinyldarkscratch) [inline static/resources/serviceworker.js:1]: Can we undo this change?  This is the only change in this file -- I'm guessing that this was the wrong file.

**Review by vinyldarkscratch — CHANGES_REQUESTED** [review]
Thanks @saschanaz, this looks good to me!  Just got a couple of changes to request.

File: static/resources/sharedworker.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Mozilla Corporation
```
Comment (saschanaz) [inline static/resources/sharedworker.js:1]: I'll wait until I figure it out about the license: https://github.com/foolip/mdn-bcd-collector/pull/548#discussion_r495488389

Or should this still be Google LLC as this is really just a copy of worker.js? 👀

Comment (vinyldarkscratch) [reply static/resources/sharedworker.js:1]: Personally I would think that the license would remain the same since the new file is a carbon copy of the existing file?  (I'm requesting a revert of this change because it's modifying the service worker file, not the new shared worked.  😛)

Comment (saschanaz) [reply static/resources/sharedworker.js:1]: Okay, I'll do that 👍

**Comment by saschanaz** [comment]
BTW, is it intentional that "Limit Exposure" works only when selecting a specific test but not when doing "All Tests"?

**Review by vinyldarkscratch — APPROVED** [review]
Awesome, thank you for the quick changes, we're now all set to go!  👍 

In response to your question about the scope limitation not working on "All Tests", it looks like you've found a bug -- the scope _should_ be limiting, but it doesn't seem like it's getting added into the URL for some odd reason.  I'm working on the views, so I'll tackle it ASAP!
